web/packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -618,3 +654,34 @@ async function createGrpcCredentials(
 
   return grpcCreds.createClientCredentials(mainProcessKeyPair, tshdCert);
 }
+
+// Must be big enough to get past through the stack trace that Node.js usually outputs when the
+// shared process fails.
+const NO_OF_LAST_LOGS_KEPT = 25;
```

Comment:
Is this arbitrary?

Reply:
As I said in the description:

> The number 25 was chosen by looking at [the logs from the glibc error](https://gravitational.slack.com/archives/C01TYKHFVTQ/p1709053320489329?thread_ts=1709051872.643429&cid=C01TYKHFVTQ) and making sure that we'd have been able to see the actual issue in the dialog.

How should I update the comment to make this more clear? I don't really want to leave a link to a Slack message. Maybe I should just copy over those glibc logs here and then link to this pull request?

Reply:
I updated the comment with a link to the PR and copied those error logs from Slack to the PR description.

Reply:
Ah sorry. I missed that in the description. I didn't mean for you to update the comment but appreciate it either way.

Reply:
Np, it's a good thing that it ended up being more clear. Otherwise how would someone reading it really know how "the stack trace that Node.js usually outputs when the shared process fails" looks like. It's way better this way.